packages/cxl-lumo-styles/scss/themes/vaadin-tabs.scss

```diff
@@ -67,77 +101,6 @@ $vaadin-tab-horizontal-padding: 0.75rem;
   margin-left: auto;
 }
 
-// Featured course slider
```

Comment:
Why it's suddenly gone? It's not about menu

Reply:
It's was in the wrong place, it belongs to cxl-tabs-slider.scss. I had already copied it over, but chose to keep this until we merged the other issue. Now it was making it harder to compare files and find the menu bug, so I cleaned it up.